api/v1/health/server/server.go

```diff
@@ -25,8 +22,11 @@ import (
 	flags "github.com/jessevdk/go-flags"
 	"golang.org/x/net/netutil"
 
+	"github.com/cilium/cilium/api/v1/health/server"
```

Comment:
I'm surprised at these changes in `api/v1/health/server/server.go` and `api/v1/server/server.go`, only the imports are changing but there's no other changes. Line 290 for instance uses the `x509` package, so this seems like a breaking change. I think we should drop the changes to these two files?

Reply:
Yeah that's a problem.. but we don't even make it that far:
```
make[1]: Leaving directory '/home/josh/go/src/github.com/cilium/cilium/cilium'
make[1]: Entering directory '/home/josh/go/src/github.com/cilium/cilium/daemon'
CGO_ENABLED=0 go build -mod=vendor -ldflags '-X "github.com/cilium/cilium/pkg/version.Version=1.7.90 f411adcbf 2020-02-25T10:34:54-08:00 go version go1.13.6 linux/amd64" -X "github.com/cilium/cilium/pkg/envoy.RequiredEnvoyVersionSHA=c31482c3e49670980c05cafc914320f7949b266f" -X "github.com/cilium/cilium/pkg/datapath/loader.DatapathSHA=5e3b6c150445abcefcc48bd0358fe1b421bb369a"'  -o cilium-agent
import cycle not allowed
package github.com/cilium/cilium/daemon
	imports github.com/cilium/cilium/cilium-health/launch
	imports github.com/cilium/cilium/pkg/health/server
	imports github.com/cilium/cilium/api/v1/health/server
	imports github.com/cilium/cilium/api/v1/health/server
import cycle not allowed
package github.com/cilium/cilium/daemon
	imports github.com/cilium/cilium/api/v1/server
	imports github.com/cilium/cilium/api/v1/server
Makefile:8: recipe for target 'cilium-agent' failed
```

Removing two more files... :grimacing: